### DISCUSSION TITLE
Add/update docs for Exclude Objects

### DISCUSSION THREAD

**Discussion by Naninani** [Guilouz/Klipper-Flsun-Speeder-Pad#10] [Ideas]
Just saw a cool vid that when printing multiple objects, allows Klipper to exclude objects while a print is in progress.
https://www.klipper3d.org/Exclude_Object.html
https://www.youtube.com/watch?v=QTwRZ_M159Q

I tried it out and it works great.

I'd be great to extend this great project and add that in the docs.

**Comment by Guilouz** [comment]
It’s already in my page in useful links.
